AwesomeTSProject/components/SingleCrypto/Index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {cryptoType} from '../../store/types/CryptoTypes';
+import {
+  OneCrypto,
+  NamesContainer,
+  CryptoName,
+  CryptoSymbol,
+  CryptoLogo,
+  ActualValue,
+  Increase,
+  Decrease,
+  IncArrow,
+  DecArrow,
+  ArrowCont,
+} from './Styles';
+
+export default function SingleCrypto({
+  name,
+  symbol,
+  value,
+  incDec,
+  imageURL,
+}: cryptoType) {
+  return (
+    <OneCrypto>
+      <CryptoLogo source={imageURL} />
+      <NamesContainer>
+        <CryptoName>{name}</CryptoName>
+        <CryptoSymbol>{symbol}</CryptoSymbol>
+      </NamesContainer>
+      <ActualValue>${value}</ActualValue>
+      {parseInt(incDec, 10) > 2.5 ? (
+        <ArrowCont>
+          <IncArrow source={require('../../assets/increaseArrow.png')} />
```

Comment:
you can import images and use it here, 

import increaseArrow from '../../assets/increaseArrow.png'